.github/workflows/R_CMD_CHECK.yml

```diff
@@ -42,8 +42,10 @@ env:
 jobs:
   R-CMD-check:
     runs-on: ${{ matrix.config.os }}
-
     name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+    branches:
+      - '**'
+      - '!feature*'
```

Comment:
@Tmonster is this the right fix?
The job seems to straight up fail on this <https://github.com/duckdb/duckdb/actions/runs/8293676283/job/22697228244?pr=10766>

And it looks like that's because the upstream is not something it's expecting
I figured if we skip it on feature, then the merge from `feature` -> `main` will trigger this so all is good?

Reply:
@krlmlr I am disabling this job from running if the PR is being made to `feature` due to the failure posted in the message above, can you fix this later and enable it again?

Reply:
The semantics of the vendoring script in duckdb-r have changed. I'll take a look.

For context: we're now polling from the main repo every 10 minutes or so, commit by commit: https://github.com/duckdb/duckdb-r/actions/workflows/vendor.yaml . I can restore the original `vendor.sh` script because this repo seems to rely on it, and use a different name for the new vendoring scheme. Open to alternative ideas.

Reply:
Should be fixed in https://github.com/duckdb/duckdb-r/commit/52468569189ecef34016d6a88c9eac0020a56668, can you please reenable and try again?